releng/platform-definitions/pom.xml

```diff
@@ -44,6 +44,7 @@
 		<jmc.config.path>${project.basedir}/../../configuration</jmc.config.path>
 	</properties>
 	<modules>
+		<module>platform-definition-2024-03</module>
```

Comment:
Should any of the older platforms be removed at this time as well? Or save that for another PR?

Reply:
I'll do it in a separate PR.